Mail/Views/New Message/ComposeMessageMacosToolbarView.swift

```diff
@@ -0,0 +1,110 @@
+/*
+ Infomaniak Mail - iOS App
+ Copyright (C) 2024 Infomaniak Network SA
+
+ This program is free software: you can redistribute it and/or modify
+ it under the terms of the GNU General Public License as published by
+ the Free Software Foundation, either version 3 of the License, or
+ (at your option) any later version.
+
+ This program is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License
+ along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import InfomaniakCoreSwiftUI
+import InfomaniakRichHTMLEditor
+import MailCore
+import MailCoreUI
+import MailResources
+import Popovers
+import SwiftUI
+
+struct ComposeMessageMacosToolbarView: View {
+    @Binding var isShowingLinkAlert: Bool
+    @Binding var isShowingFileSelection: Bool
+
+    @ObservedObject var textAttributes: TextAttributes
+
+    private let extras: [EditorToolbarAction] = [
+        .attachment,
+        .link
+    ]
+
+    private let textFormats: [EditorToolbarAction] = [
+        .bold,
+        .underline,
+        .italic,
+        .strikeThrough,
+        .cancelFormat
+    ]
+
+    private let textItems: [EditorToolbarAction] = [
+        .unorderedList
+    ]
+
+    private var allItems: [[EditorToolbarAction]] {
+        [extras, textFormats, textItems]
+    }
+
+    var body: some View {
+        VStack(spacing: 0) {
+            HStack(spacing: IKPadding.extraSmall) {
+                ForEach(allItems, id: \.self) { items in
+                    ForEach(items, id: \.self) { item in
+                        Button {
+                            item.action(
+                                textAttributes: textAttributes,
+                                isShowingLinkAlert: &isShowingLinkAlert,
+                                isShowingFileSelection: &isShowingFileSelection
+                            )
+                        } label: {
+                            item.icon.swiftUIImage
+                                .resizable()
+                                .frame(width: 16, height: 16)
+                                .padding(value: .small)
+                        }
+                        .buttonStyle(MacosToolbarButtonStyle(isActive: item.isSelected(textAttributes: textAttributes)))
```

Comment:
Make a static style func so that it can be called like this `.buttonStyle(.macosToolbarButtonStyle(...))`